### ISSUE TITLE
Kadira.trackError(type, message); - exception and not message

### ISSUE THREAD

**Issue by dotnetwise** [meteorhacks/kadira#150]
Can you actually call `Kadira.trackError(type, exception);` where exception is a new trapped exception or `new Error('some message')`;

The reason for this is to keep the original stacktrace

**Comment by arunoda** [comment]
For the second arguments, we are accepting an string. So, it could be the
stack.
You can get it from `error.stack`

**Comment by dotnetwise** [comment]
Yes, but it's not the same as `throw new Exception("msg")` since you already have the trace mechanism in `kadira`.

Please detect if it's not a `string` but  rather a javascript error object and use it accordingly. Thanks

**Comment by mjgallag** [comment]
https://support.kadira.io/knowledgebase/articles/421165-tracking-custom-errors

Yes, it would be awesome if instead of having to do this:
```javascript
try {
  // your code 
} catch(ex) {
  // handle error
  var type = 'my-error-type';
  var message = ex.message;
  Kadira.trackError(type, message);
}
```
We could do this:
```javascript
try {
  // your code 
} catch(ex) {
  // handle error
  Kadira.trackError(ex);
}
```
:+1: for this feature request
